docs/source/commands/init.mdx

```diff
@@ -16,13 +16,64 @@ Running `rover init` starts a short wizard that helps you create a new graph.
 rover init
 ```
 
+<ExpansionPanel title="Experiencing issues?">
+
+    - If you experience a crash, run `rover config clear` and then run `rover config auth` before running `rover init` again.
+
+    - If you receive `error: Invalid API key found.` unset the `APOLLO_KEY` environment variable:
+
+        <Tabs>
+            <Tab label="Linux / MacOS">
+
+            ```terminal showLineNumbers=false
+            unset APOLLO_KEY
+            ```
+
+            </Tab>
+            <Tab label="Windows">
+            #### Powershell
+
+            ```terminal
+            Remove-Item Env:APOLLO_KEY
+            ```
+
+            #### Command Prompt
+            ```terminal
+            set APOLLO_KEY=
+            ```
+
+            </Tab>
+        </Tabs>
+
+     Then, run `rover config auth` before running `rover init` again.
+
+</ExpansionPanel>
+
 The wizard walks you through a set of questions to help you choose the best option for your use case. When you're done, you'll have:
 
 - A new graph in GraphOS
 - A set of local configuration files and boilerplate code that represent the graph 
 - Credentials to interact with GraphOS
     - These credentials are necessary for actions like [developing your graph locally](/rover/commands/dev) or [publishing graph updates](/graphos/platform/schema-management/delivery/publish).
 
+#### Options
+
```

Comment:
Ah, here it would be good to note that a user will only entirely skip the wizard if they pass all flags. Otherwise, they will still go through the flow to complete the remaining steps that they didn't pass a flag argument for 👍